src/components/Input/SearchInput/SearchInput.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import Search from '@material-design-icons/svg/round/search.svg';
+import Clear from '@material-design-icons/svg/round/cancel.svg';
+
+import { Input, Props } from '../Input';
+
+export const SearchInput = React.forwardRef<HTMLInputElement, Props>(
+    (
+        {
+            id,
+            children,
+            context = undefined,
+            disabled = false,
+            readOnly = false,
+            isBlock = false,
+            size = 'medium',
+            value: initialValue = '',
```

Comment:
What do you mean exactly? Try the SearchInput? It's a component in input

Reply:
how can I set `initialValue` in storybook? I wanted to test if this assignment here won't make a bug. Like if you write something and then clear the field, will it be empty?

Reply:
InitialValue takes the value of 'value', unless value is undefined, then it becomes ''. As it isn't part of the props, you can't set it in storybook, just like you can't set it for input, without simply typing it. I don't know if that should be a possibility? Then I need to add value to the props. 

I used this method so I could then create a state of value, setValue and directly use that in the input.

Reply:
Input has the following props: `interface BaseProps extends Omit<React.InputHTMLAttributes<HTMLInputElement>, 'size'> {...}`. So it has all the native attributes possible, including value. This way you can implement a controlled component. These are passed to Input correctly via `rest` that is applied to the `<input>` 
I'm a bit confused by the code, and I wonder how it will work if someone implements the component as controlled. Maybe you could add a story where you pass `value`. But even if not, we will be sure that it is possible to implement and it works as expected

Reply:
Tell me about it, haha. I'm also confused. But I can't keep SearchInput as a controlled component, right? I wouldn't know how to, with the icons influencing the value...

Reply:
hhhmmm... In JF we almost always use controlled components, since we need to do something with the value, like filter results that we show and so on. Once we have it in the code, it easier to use controlled components.

Maybe you can think of an other approach, and use ref on input to see if it has a value or not, and set the states based on that? Would that work?